installers/dhall/linux.dhall

```diff
@@ -1,25 +1,34 @@
--- XXX: deal with XDG_DATA_HOME generalisation -- pass dataDir as an argument?
-let dataDir = "$HOME/.local/share/Daedalus/"
+\(cluster : ./cluster.type)      ->
+let dataDir = "''${HOME}/.local/share/Daedalus/"
+    -- XXX: deal with XDG_DATA_HOME being an actual variable -- pass dataDir as an argument?
+    --    ..or, maybe installation data is a better solution?
 in
-{ name      = "linux"
-, nodeArgs  =
-  { keyfile          = dataDir ++ "Secrets-1.0/secret.key"
-  , logsPrefix       = dataDir ++ "Logs"
-  , updateLatestPath = dataDir ++ "FIXME"
-  , walletDBPath     = dataDir ++ "Wallet-1.0"
+{ name      = "macos64"
+    -- XXX: this is an ugly workaround for the lack of a linux configuration key in configuration.yaml
+, configurationYaml  = "''${DAEDALUS_CONFIG}/configuration.yaml"
+, nodeArgs           =
+  { keyfile          = "Secrets/secret.key"
+  , logsPrefix       = "Logs"
+  , topology         = "''${DAEDALUS_CONFIG}/wallet-topology.yaml"
+  , updateLatestPath = "${dataDir}/${cluster.name}/installer.sh"
+  , walletDBPath     = "Wallet/"
   }
 , pass      =
-  { nodePath            = "./cardano-node"
-  , nodeDbPath          = dataDir ++ "DB-1.0"
-  , nodeLogPath         = dataDir ++ "Logs/cardano-node.log"
+  { nodePath            = "''${DAEDALUS_BRIDGE}/bin/cardano-node"
+  , nodeDbPath          = "DB/"
+  , nodeLogConfig       = "''${DAEDALUS_CONFIG}/daedalus.yaml"
+  , nodeLogPath         = "''${HOME}/.local/share/Daedalus/${cluster.name}/Logs/cardano-node.log"
```

Comment:
this path will depend on the value of `$XDG_DATA_DIR`

Reply:
Yes, I'm handling that soon..

Reply:
Done.